### DISCUSSION TITLE
How many external displays can we use in SEB in Windows OS 10 & 11?

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#628] [Q&A]
Hi Team,

We are using SEB version 3.5.0 on Windows 10 OS.

We have configured ""Maximum allowed number of connected displays = 3"" in the SEB Security Configuration & We have enabled "Ignore errors when validating display configuration". Because, We are connecting the laptop with 2 more monitors. So, Totally 3 displays.

""Scenario 1: (Extended display)""
First we have extended the display in Windows Display settings and launched the SEB. Once launched the SEB, SEB is showing on 1st screen and the extended 2nd screen is also showing. But the 3rd display screen is in OFF mode. Why the 3rd screen is not showing as extended display? why it turned off?

""Scenario 2: (Duplicate display)""
Now, we have duplicated the display in Windows Display settings and launched the SEB. Once launched the SEB, SEB showing 1st and 3rd screens are duplicate displays and the 2nd screen is showing as extended display. why the 2nd screen is not showing as duplicate display instead of extended display?

Kindly let us know why the 3rd & 2nd screen is showing like that on the above scenarios. Do we need to change anything in the configuration?

Awaiting for your response.

Here I attached the SEB Configuration screenshot for your reference.

![Display_3](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/111885140/bf1bc527-2b4c-4c8b-a474-2ca6cef21017)

**Comment by Chris251290** [comment]
Hi Team, Can you please response for this query?

So that we can modify if any configurations needed.

**Comment by dbuechel** [comment]
Theoretically, SEB should just use whatever display configuration you've configured in Windows (i.e. SEB will show its UI on the main display which you've configured in Windows). With respect to mirroring, I'd expect this to work as intended, however extending a desktop is currently not supported by SEB, i.e. SEB will always and only render its UI on the main display. There actually is a feature request related to this question, see #564.

**Reply by Chris251290** [reply]
Hi Damian, Thanks for the clarification.
